scripts/scheduled-task-commands/refreshOrgchartEmployees.php

```diff
@@ -1,62 +1,23 @@
 <?php

```

Comment:
This has a lot going on, we just pushed a change for this it looks like everything is there but just want to call that out.  The LEAFNexusAPI is another one that had some changes and look to be correct.